### PR TITLE
Lab 2 Work: Reworked App.js; Basic work on todoList; Imported Vector Graphics for BottomNavigationTab

### PR DESCRIPTION
**Pull request by BJladika**

### CODE REVIEW

File: novgodorovMaxim/screens/useStateLab.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <Text 
+                style={{
+                    textAlign:'center',
+                    marginTop:25
+                }}>
+                Lab2: Hello BJladika! v3
+            </Text>
+      
+            <TouchableOpacity
+                style={{
```
Comment (arri1) [inline novgodorovMaxim/screens/useStateLab.js]: StyleSheets

File: novgodorovMaxim/components/todoListItem.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    style={{
+                        flexDirection: 'row',
+                        width: '100%',
+                    }}
+                >
+                    <Image
+                        source={{
+                            uri: 'https://raw.githubusercontent.com/AboutReact/sampleresource/master/old_logo.png',
+                        }}
+                        style={{
```
Comment (arri1) [inline novgodorovMaxim/components/todoListItem.js]: перевести в  styles

File: novgodorovMaxim/screens/useStateLab.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        <SafeAreaView>
+            <Text style={styles.text_lab_title}>
+                Lab2: Hello BJladika! v4
+            </Text>
+      
+            <TouchableOpacity
+                style={styles.tch_opacity}
+                title={'Press X to Win'}
+                onPress={() => {
+                setColor(colorArray[color_int])
```
Comment (arri1) [inline novgodorovMaxim/screens/useStateLab.js]: тут с форматированием проблемы

File: novgodorovMaxim/screens/useStateLab.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+const UseStateLab = () => {
+    const [color, setColor] = useState('white')
+    const [count,setCount] = useState(0)
+    const [color_int,setCountNumb] = useState(0)
+    
+    useEffect(()=>{
+        setCount(count+1)
+    },[color])
+
+    const styles = StyleSheet.create({
```
Comment (arri1) [inline novgodorovMaxim/screens/useStateLab.js]: вывести styles за UseStateLab

**Comment by BJladika** [comment]
testing close, to pull current project because can't pull it with open pull request

**Comment by BJladika** [comment]
Reopened, because pulled finally from the original project

File: novgodorovMaxim/screens/useStateLab.js (hunk 4 above)
Comment (arri1) [reply novgodorovMaxim/screens/useStateLab.js]: почему не вывел?

Comment (BJladika) [reply novgodorovMaxim/screens/useStateLab.js]: упс забыл, ща выведу

**Pull request merged**